### ISSUE TITLE
Can't enable in Blender because of error in shader

### ISSUE THREAD

**Issue by Unesty** [eliemichel/DagAmendment#1]
ERROR (gpu.shader): pyGPUShader VertShader:
      |
   32 |         vec4 color = texelFetch(jbuffermaps[param], coord, 0);
      |                           ^
      | Error:  sampler arrays indexed with non-constant expressions are forbidden in GLSL 1.30 and later

Traceback (most recent call last):
  File "/usr/share/blender/2.93/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/__init__.py", line 84, in <module>
    from . import overlays
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/overlays.py", line 29, in <module>
    from .shaders import line_shader, line_shader_2d, point_shader
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/shaders.py", line 252, in <module>
    jbuffer_reduce_shader = gpu.types.GPUShader('''
Exception: Shader Compile Error, see console for more details

**Comment by eliemichel** [comment]
Hi @Unesty thanks for your feedback.

This error had been fixed by 7802395 but was indeed not available in the last published release, so here is a new one that includes the bugfix: https://github.com/eliemichel/DagAmendment/releases/tag/v1.0.1

**Comment by Unesty** [comment]
@eliemichel For some reason, same here. And I used release and then built from source.

**Comment by eliemichel** [comment]
My bad, for some reason the commit got reverted. I just updated the release.

**Comment by Unesty** [comment]
Now this
```
Read prefs: /bunny/.config/blender/2.93/config/userpref.blend
/run/user/1004/gvfs/ non-existent directory
Traceback (most recent call last):
  File "/usr/share/blender/2.93/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/__init__.py", line 84, in <module>
    from . import overlays
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/overlays.py", line 29
    from .shaders import line_shader,, point_shader
                                     ^
SyntaxError: invalid syntax
```

**Comment by Unesty** [comment]
After fixing this error new appears
```
reloading addon: DagAmendment 1633701810.674106 1633702126.534784 '/bunny/.config/blender/2.93/scripts/addons/DagAmendment/__init__.py'
Modules Installed () from '/bunny/Downloads/DagAmendment/releases/DagAmendment-v1.0.1.zip' into '/bunny/.config/blender/2.93/scripts/addons'
Traceback (most recent call last):
  File "/usr/share/blender/2.93/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/__init__.py", line 85, in <module>
    from . import tools
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/tools.py", line 29, in <module>
    from .draw_utils import draw_lines_2d
  File "/bunny/.config/blender/2.93/scripts/addons/DagAmendment/draw_utils.py", line 29, in <module>
    from .shaders import tex2points_shader
ImportError: cannot import name 'tex2points_shader' from 'DagAmendment.shaders' (/bunny/.config/blender/2.93/scripts/addons/DagAmendment/shaders.py)
```

**Comment by haxxolotl** [comment]
```
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.93\2.93\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\__init__.py", line 84, in <module>
    from . import overlays
  File "C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\overlays.py", line 29
    from .shaders import line_shader,, point_shader
                                     ^
SyntaxError: trailing comma not allowed without surrounding parentheses

```
same thing here

**Comment by haxxolotl** [comment]
After resolving the extra comma issue, getting the following:
```
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.93\2.93\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\__init__.py", line 85, in <module>
    from . import tools
  File "C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\tools.py", line 29, in <module>
    from .draw_utils import draw_lines_2d
  File "C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\draw_utils.py", line 29, in <module>
    from .shaders import tex2points_shader
ImportError: cannot import name 'tex2points_shader' from 'DagAmendment.shaders' (C:\Users\enen9\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\DagAmendment\shaders.py)


```

**Comment by haxxolotl** [comment]
Removing that line allows addon to be successfully added
`from .shaders import tex2points_shader`

**Comment by eliemichel** [comment]
So sorry, working a bit in a hurry on this, I had not taken the time to test my release. :/
Fixed again in [v1.0.1](https://github.com/eliemichel/DagAmendment/releases/tag/v1.0.1), thank you both of you!
